scripts/crawl_expand.py

```diff
@@ -84,6 +83,7 @@ def makeNewStatusTable(config, updatedProviders, placeProviderIDs, newProviders)
 
 if __name__ == '__main__':
     TEST_CONFIG = { "yelp3": 3,
-                    "tripadvisor": 3 }
+                    "tripadvisor": 3,
+                    "google": 3 }
 
-    expandPlaces(TEST_CONFIG, GPS_LOCATIONS["CHICAGO_CENTER"], 3)
+    expandPlaces(TEST_CONFIG, GPS_LOCATIONS["CHICAGO_CENTER"], 30)
```

Comment:
Is committing 3 -> 30 intentional? (I'm not sure over how large an area we should be searching)